backend/inspirehep/accounts/api.py

```diff
@@ -37,15 +40,19 @@ def is_loggedin_user_email(email):
 
 
 def get_current_user_orcid():
-    try:
-        orcid = (
-            UserIdentity.query.filter_by(id_user=current_user.get_id(), method="orcid")
-            .one()
-            .id
-        )
-        return orcid
-    except NoResultFound:
-        return None
+    if current_user:
```

Comment:
the current_user will be always set even if the user is not logged (as Anonymous) we you would like to add this extra if here?

Reply:
@MJedr ``current_user`` is a redundant check, why want it here?

Reply:
@drjova yes, you're right. It's updated :)